firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -69,30 +69,30 @@ void homeAxis(int motor[])
   currentMotorNumEighthSteps *= -1;
   while (digitalRead(motor[ENDSTOP_PIN]) == LOW)
   {
+    // Moves motor
     digitalWrite(motor[STEP_PIN], LOW);
     delay(1);
     digitalWrite(motor[STEP_PIN], HIGH);
 
-    // Motor is moved above, current position for whatever motor updated here
+    // Updates current motor position
     *currentMotorPos += currentMotorNumEighthSteps;
   }
 
-  // Flipping direction again to move motor away from home to prepare for fine-tuning
+  // Flips direction again to move motor away from home to prepare for fine-tuning
   digitalWrite(motor[DIR_PIN], RIGHT);
   currentMotorNumEighthSteps *= -1;
```

Comment:
Same as above, this should be setting equal to the expected value of 1 instead of essentially multiplying by -1 twice